org.eclipse.m2e.pde.ui/src/org/eclipse/m2e/pde/ui/editor/messages.properties

```diff
@@ -18,9 +18,11 @@ MavenTargetLocationWizard_12=Artifacts are resolved against the workspace <a hre
 MavenTargetLocationWizard_14=Label (optional)
 MavenTargetLocationWizard_9=Missing OSGi-Manifest
 MavenTargetLocationWizard_10=Dependencies scope
+MavenTargetLocationWizard_17=Dependencies depth
```

Comment:
What do you think about "Include dependencies for depth"  instead?
This proposal is IMHO not yet ideal but would be fit into the schema of my other proposal. Maybe you have a better one.

Reply:
Same here ...